tools/pot/docs/BestPractices.md

```diff
@@ -12,43 +12,43 @@
 
-> **NOTE**: POT uses inference on the CPU during model optimization. It means the ability to infer the original
-> floating-point model is a prerequisite for model optimization. 
-> It is also worth mentioning that in the case of 8-bit quantization it is recommended to run POT on the same CPU
+> **NOTE**: POT uses inference on the CPU during model optimization. It means that ability to infer the original
+> floating-point model is essential for model optimization. 
+> It is also worth mentioning that in case of the 8-bit quantization, it is recommended to run POT on the same CPU
 > architecture when optimizing for CPU or VNNI-based CPU when quantizing for a non-CPU device, such as GPU, VPU, or GNA.
 > It should help to avoid the impact of the [saturation issue](@ref pot_saturation_issue) that occurs on AVX and SSE based CPU devices. 
 
 ## Improving accuracy after the Default Quantization
-Parameters of the Default Quantization algorithm with basic settings are shown below:
+Parameters of the Default Quantization algorithm with basic settings are presented below:
 ```python
 {
     "name": "DefaultQuantization", # Optimization algorithm name
     "params": {
         "preset": "performance", # Preset [performance, mixed] which controls 
                                  # the quantization scheme. For the CPU: 
-                                 # performance - symmetric quantization  of weights and activations
-                                 # mixed - symmetric weights and asymmetric activations
+                                 # performance - symmetric quantization  of weights and activations.
+                                 # mixed - symmetric weights and asymmetric activations.
         "stat_subset_size": 300  # Size of subset to calculate activations statistics that can be used
-                                 # for quantization parameters calculation
+                                 # for quantization parameters calculation.
     }
 }
 ```
 
-In the case of substantial accuracy degradation after applying this method there are two alternatives:
-1.  Hyperparameters tuning
-2.  AccuracyAwareQuantization algorithm
+There are two alternatives in case of substantial accuracy degradation after applying this method:
+1.  Hyperparameters tuning.
+2.  AccuracyAwareQuantization algorithm.
 
 ### Tuning Hyperparameters of the Default Quantization
 The Default Quantization algorithm provides multiple hyperparameters which can be used in order to improve accuracy results for the fully-quantized model. 
 Below is a list of best practices that can be applied to improve accuracy without a substantial performance reduction with respect to default settings:
-1.  The first recommended option is to change the `preset` from `performance` to `mixed`. This enables asymmetric quantization of 
+1.  The first recommended option is to change the *preset* from `performance` to `mixed`. This enables asymmetric quantization of 
```

Comment:
I think it was to highlight that "preset" value is the one to change. But maybe instead of using italic, let's change it to that:

```suggestion
1.  The first recommended option is to change the **preset** from `performance` to `mixed`. This enables asymmetric quantization of 
```

Reply:
I would be cautious, the preset here may be a parameter, in which case the code formatting would fit.